src/main/java/pl/pop/interview/master/question/QuestionFacade.java

```diff
@@ -42,11 +56,10 @@ public QuestionDTO findRandomQuestion() {
     }
 
     public Question getQuestion(Long questionId) {
-
         return questionRepository
-                .findById( questionId )
+                .findById(questionId)
                 .orElseThrow(
-                        () -> new NotFoundException( "Question with ID " + questionId + " does not exist!" )
+                        () -> new NotFoundException("Question with ID " + questionId + " does not exist!")
```

Comment:
Ja wiem, że to jest najbardziej intuicyjne, ale nie chcemy pokazywać ID pytania dalej. 

Jeżeli takie informacje wyjdą na zewnątrz, to w zasadzie naruszymy pewien rodzaj enkapsulacji. 

"Question with provided ID cannot be find" + do tego można zalogować to w RestControllerAdvice, o ile damy taką informację. 

Ale sama wiadomość, która często wychodzi wyżej, nie powinna wyjść na UI w żadnym wypadku.

Reply:
Czy chodzi o to żeby sam komunikat zmienić na "question not found"?

Reply:
ok, done